data/human/human missions.txt

```diff
@@ -4925,3 +4925,146 @@ mission "Riley"
 			label end
 			`	You say your goodbyes, leaving to your ship and returning to the stars.`
 				decline
+
+
+
+mission "Syndicate Business 1"
+	name "Rescue Kidnapped Man in <waypoints>"
+	description "A mysterious man has been kidnapped by a heavy pirate warship in <waypoints>. Destroy the ship to rescue him."
+	minor
+	to offer
+		random < 40
+		credits > 5000000
+	source 
+		government Syndicate
+		attributes "core"
+	waypoint "Moktar"
+	destination "Sunracer"
+	on offer
+		conversation
+			`Returning to your ship after an uneventful dinner at the spaceport, your ship's communication screen blares an alarm as the words "EMERGENCY DISTRESS SIGNAL" flash in red. A terrified-looking man in a suit appears on the screen. "I don't have much time - I've been kidnapped. If you're receiving this message, I need you to destroy the ship. I've managed to barricade myself in an escape pod."` 
```

Comment:
How is the player getting this signal? How is the man sending out the signal? Why doesn't anybody else get the signal? How does he manage to hold out for over a day?